### PR TITLE
[Feature] Draggle & Droppable Emoji Categories

### PR DESCRIPTION
**Pull request by dclstn**
QA Testing Checklist

- [x] Twitch Dark Mode Chat + Emote Menu
- [x] Twitch Light Mode Chat + Emote Menu
- [x] Twitch Dark Mode Settings Menu (from Chat Settings + from User Dropdown)
- [x] Twitch Light Mode Settings Menu (from Chat Settings + from User Dropdown)
- [x] Twitch POPOUT Dark Mode Chat + Emote Menu + Chat Settings
- [x] Twitch POPOUT Light Mode Chat + Emote Menu + Chat Settings
- [x] Twitch Moderation Chat Dark Mode Chat + Emote Menu + Chat Settings
- [x] Twitch Moderation Chat Light Mode Chat + Emote Menu + Chat Settings
- [x] Twitch Dashboard Chat Dark Mode Chat + Emote Menu + Chat Settings
- [x] Twitch Dashboard Chat Light Mode Chat + Emote Menu + Chat Settings
- [x] YouTube Dark Mode Chat + Emote Menu
- [x] YouTube Light Mode Chat + Emote Menu
- [x] YouTube Dark Mode Settings Menu (from Chat Settings)
- [x] YouTube Light Mode Settings Menu (from Chat Settings)
- [x] YouTube POPOUT Dark Mode Chat + Emote Menu + Chat Settings
- [x] YouTube POPOUT Light Mode Chat + Emote Menu + Chat Settings
- [ ] YouTube Studio Dark Mode Chat + Emote Menu + Chat Settings

### CODE REVIEW

File: src/modules/emote_menu/components/EmoteMenuPopover.module.css [hunk 1]
```diff
@@ -1,5 +1,5 @@
 .popover {
-  height: 410px;
+  height: 402px;
```
Comment (night) [inline src/modules/emote_menu/components/EmoteMenuPopover.module.css:2]: why is it smaller by 8px?

File: src/modules/emote_menu/components/Sidebar.module.css [hunk 2]
```diff
@@ -1,33 +1,108 @@
...
 
-.sidebar::-webkit-scrollbar {
+.content::-webkit-scrollbar {
   display: none;
 }
 
 .active {
   background-color: var(--bttv-rs-bg-card);
+  border-right: 2px solid var(--bttv-rs-navs-selected);
+  padding-right: 10px !important;
```
Comment (night) [inline src/modules/emote_menu/components/Sidebar.module.css:26]: why do we need !important here?

File: src/modules/emote_menu/components/Sidebar.module.css [hunk 3]
```diff
@@ -1,33 +1,108 @@
...
 }
 
-.navItemContent {
+.dragging {
+  border-radius: 4px;
+  color: var(--bttv-rs-text-heading);
+  display: flex;
+  background-color: unset !important;
+  border-right: unset !important;
+  padding: 6px 12px !important;
```
Comment (night) [inline src/modules/emote_menu/components/Sidebar.module.css:52]: why do these need !important?

File: src/modules/emote_menu/components/EmoteMenuPopover.module.css (hunk 1 above)
Comment (dclstn) [reply src/modules/emote_menu/components/EmoteMenuPopover.module.css:2]: Preview.jsx height was reduced to 48px

**Review by night — APPROVED** [review]

**Pull request merged**